editions/tw5.com/tiddlers/howtos/How to export tiddlers.tid

```diff
@@ -0,0 +1,30 @@
+created: 20151006145934884
+modified: 20151006155211988
+tags: Learning
```

Comment:
noted that too and I agree, so maybe ImportTiddlers should be renamed while [some redirecting](http://tobibeer.github.io/tb5/#Aliases%20And%20Redirecting%20Dead%20Links) could take care of pointing it to "Import Tiddlers"